apps/mail/components/mail/mail.tsx

```
@@ -22,20 +22,8 @@ import { useRouter } from "next/navigation";
-    email: string;
-    icon: ReactNode;
-  }[];
-  folder: string;
-  defaultLayout: number[] | undefined;
-  defaultCollapsed?: boolean;
-  navCollapsedSize: number;
-  muted?: boolean;
-}
-
-export function Mail({ folder }: MailProps) {
+export function Mail() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Mail'. Consider to delete it or rename it.

'Mail' is defined here:


(lint/suspicious/noRedeclare)